puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml

```diff
@@ -60,7 +60,12 @@
           values:
             - fast
     execution-strategy:
-      combination-filter: "!(foreman_branch == 'develop' && ruby == '1.9.3')"
+      combination-filter: >
+        "(ruby == '1.9.3' && database == 'postgresql') ||
```

Comment:
The foreman_branch check is required here, it should only be run when the branch is older than develop (or remove this capability if no plugin is using it) as develop doesn't support 1.9.3.